tests/test_image_manager.py

```diff
@@ -9,83 +10,116 @@
+    PNG_TO_UPLOAD = "images/test.png"
 
     def setUp(self):
         aws_connector = AwsConnector()
         self.s3_resource = aws_connector.get_s3_resource()
-        self.s3_resource.create_bucket(Bucket=self.bucket, CreateBucketConfiguration={
+        self.s3_resource.create_bucket(Bucket=self.BUCKET, CreateBucketConfiguration={
             'LocationConstraint': 'eu-west-2'},)
-        self.image_manager = ImageManager(self.s3_resource, self.bucket)
+        self.image_manager = ImageManager(self.s3_resource, self.BUCKET)
 
     def tearDown(self):
-        bucket = self.s3_resource.Bucket(self.bucket)
+        bucket = self.s3_resource.Bucket(self.BUCKET)
         for key in bucket.objects.all():
             key.delete()
         bucket.delete()
 
-    def test_add_image(self):
-        # arrange
-        id = helper.get_id()
-        vector_img = self.image_manager._get_img_path(self.FILENAME_TO_UPLOAD, id)
-        png_img = self.image_manager._get_img_path(self.FILENAME_TO_UPLOAD, id, False)
-
+    def test_upload_vector_file(self):
         # act
-        self.image_manager.add_image(self.FILENAME_TO_UPLOAD, id, True)
-        self.image_manager.add_image(self.FILENAME_TO_UPLOAD, id, False)
+        vector_path = self.image_manager.upload_vector_file(
+            self.VECTOR_TO_UPLOAD, self.ITEM_ID_1)
         images = tuple(
             img.key for img in self.image_manager.bucket.objects.all())
 
         # assert
-        self.assertEqual(len(images), 2)
-        self.assertIn(vector_img, images)
-        self.assertIn(png_img, images)
-
-    def test_add_vector_image(self):
-        # arrange
-        id = helper.get_id()
-        vector_img = self.image_manager._get_img_path(self.FILENAME_TO_UPLOAD, id)
+        self.assertIn(vector_path, images)
 
+    def test_upload_png_file(self):
         # act
-        self.image_manager.add_vector_image(self.FILENAME_TO_UPLOAD, id)
+        png_path = self.image_manager.upload_png_file(
+            self.VECTOR_TO_UPLOAD, self.ITEM_ID_1)
         images = tuple(
             img.key for img in self.image_manager.bucket.objects.all())
 
         # assert
-        self.assertIn(vector_img, images)
+        self.assertIn(png_path, images)
 
-    def test_add_png_image(self):
+    def test_delete_file(self):
         # arrange
-        id = helper.get_id()
-        png_img = self.image_manager._get_img_path(self.FILENAME_TO_UPLOAD, id, False)
+        vector_path = self.image_manager.upload_vector_file(
+            self.VECTOR_TO_UPLOAD, self.ITEM_ID_1)
 
         # act
-        self.image_manager.add_png_image(self.FILENAME_TO_UPLOAD, id)
+        return_message = self.image_manager.delete_file(vector_path)
         images = tuple(
             img.key for img in self.image_manager.bucket.objects.all())
 
         # assert
-        self.assertIn(png_img, images)
+        self.assertTrue(return_message)
+        self.assertNotIn(vector_path, images)
+
+    def test__get_img_path(self):
```

Comment:
double underscore? Is that convention?

Reply:
as discussed it's because the naming on the actual functions, but I updated to single underscore as it's easy to be confused